contrib/opencensus-ext-stackdriver/tests/test_stackdriver_stats.py

```diff
@@ -115,20 +124,23 @@ def test_constructor_param(self):
         self.assertEqual(exporter.options.project_id, project_id)
 
     def test_blank_project(self):
-        self.assertRaises(ValueError, stackdriver.new_stats_exporter,
-                          stackdriver.Options(project_id=""))
+        with patch_sd_transport():
```

Comment:
Ideally the `PeriodicTask` would shut itself down gracefully at the end of the test and we wouldn't have to patch tests that don't rely on the transport.

Reply:
This was as easy as making the class a daemon thread, fixed in d693cf8.